projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy_optimise.php

```diff
@@ -546,393 +586,422 @@ static function dissolve_4value_shorthands($property, $value) {
 		$font_prop_default = & $GLOBALS['csstidy']['font_prop_default'];
-		$font_weight = array('normal', 'bold', 'bolder', 'lighter', 100, 200, 300, 400, 500, 600, 700, 800, 900);
-		$font_variant = array('normal', 'small-caps');
-		$font_style = array('normal', 'italic', 'oblique');
-		$important = '';
-		$return = array('font-style' => null, 'font-variant' => null, 'font-weight' => null, 'font-size' => null, 'line-height' => null, 'font-family' => null);
-
-		if (csstidy::is_important($str_value)) {
+		$font_weight       = array( 'normal', 'bold', 'bolder', 'lighter', 100, 200, 300, 400, 500, 600, 700, 800, 900 );
+		$font_variant      = array( 'normal', 'small-caps' );
+		$font_style        = array( 'normal', 'italic', 'oblique' );
+		$important         = '';
+		$return            = array(
+			'font-style'   => null,
+			'font-variant' => null,
+			'font-weight'  => null,
+			'font-size'    => null,
+			'line-height'  => null,
+			'font-family'  => null,
+		);
+
+		if ( csstidy::is_important( $str_value ) ) {
 			$important = '!important';
-			$str_value = csstidy::gvw_important($str_value);
+			$str_value = csstidy::gvw_important( $str_value );
 		}
 
-		$have['style'] = false;
+		$have['style']   = false;
 		$have['variant'] = false;
-		$have['weight'] = false;
-		$have['size'] = false;
-		// Detects if font-family consists of several words w/o quotes
+		$have['weight']  = false;
+		$have['size']    = false;
+		// Detects if font-family consists of several words w/o quotes.
 		$multiwords = false;
 
-		// Workaround with multiple font-family
-		$str_value = csstidy_optimise::explode_ws(',', trim($str_value));
-
-		$str_value[0] = csstidy_optimise::explode_ws(' ', trim($str_value[0]));
-
-		for ($j = 0; $j < count($str_value[0]); $j++) {
-			if ($have['weight'] === false && in_array($str_value[0][$j], $font_weight)) {
-				$return['font-weight'] = $str_value[0][$j];
-				$have['weight'] = true;
-			} elseif ($have['variant'] === false && in_array($str_value[0][$j], $font_variant)) {
-				$return['font-variant'] = $str_value[0][$j];
-				$have['variant'] = true;
-			} elseif ($have['style'] === false && in_array($str_value[0][$j], $font_style)) {
-				$return['font-style'] = $str_value[0][$j];
-				$have['style'] = true;
-			} elseif ($have['size'] === false && (is_numeric($str_value[0][$j][0]) || $str_value[0][$j][0] === null || $str_value[0][$j][0] === '.')) {
-				$size = csstidy_optimise::explode_ws('/', trim($str_value[0][$j]));
+		// Workaround with multiple font-family.
+		$str_value = self::explode_ws( ',', trim( $str_value ) );
+
+		$str_value[0] = self::explode_ws( ' ', trim( $str_value[0] ) );
+
+		for ( $j = 0; $j < count( $str_value[0] ); $j++ ) { // phpcs:ignore Squiz.PHP.DisallowSizeFunctionsInLoops.Found, Generic.CodeAnalysis.ForLoopWithTestFunctionCall.NotAllowed
+			if ( $have['weight'] === false && in_array( $str_value[0][ $j ], $font_weight ) ) { // phpcs:ignore WordPress.PHP.StrictInArray.MissingTrueStrict
```

Comment:
Just make the integer values into strings on line 853. `$str_value[0][ $j ]` is always going to be a string.